### PR TITLE
Add downloadjs@1.4.6 /w git auto-update

### PR DESCRIPTION
**Pull request by clairetsai818**
Pull request for issue: #10057 
Related issue(s): 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/rndme/download
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 251
   - Count of watchers: 18
   - Count of forks: 56

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [ ] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
@clairetsai818 there is no content in your PR, I think you should review/double check your own PR before asking the others ...

image

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
@clairetsai818 you didn't rebase with your master branch, you just merge and merge again, maybe you should add `--rebase` with your `pull` command to fix the messed up branch.

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review it, thanks.

File: ajax/libs/downloadjs/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "version": "1.4.6",
+  "description": "file downloading using client-side javascript",
+  "filename": "download.min.js",
+  "keywords": [
+    "files",
+    "dataURL",
+    "blob",
+    "download"
+  ],
+  "homepage": "http://danml.com/download.html",
```
Comment (PeterDaveHello) [inline ajax/libs/downloadjs/package.json]: /NkedM/download.html not available

**Comment by clairetsai818** [comment]
@PeterDaveHello Please help me review it, thanks.

**Comment by PeterDaveHello** [comment]
@clairetsai818 you should tell me what have you done before asking for review.

**Comment by PeterDaveHello** [comment]
You should check your checklist.

[ ] Auto-update setup -> uncheck
[x] Auto-update target/source is valid. -> checked
[x] Auto-update filemap is correct. -> checked

Strange ...

**Comment by clairetsai818** [comment]
@PeterDaveHello I've revised the checklist. Thanks!

**Pull request merged**